xcode-compat/src/main/kotlin/org/jetbrains/kotlin/xcodecompat/XcodeCompatPlugin.kt

```diff
@@ -28,10 +29,23 @@ open class KotlinXcodeExtension(private val project: Project) {
         val buildType = NativeBuildType.valueOf(System.getenv("CONFIGURATION")?.toUpperCase()
                 ?: "DEBUG")
         if (this.buildType == buildType) {
+            var dsymTask: Sync? = null
+
+            if (buildType == NativeBuildType.DEBUG) {
+                dsymTask = project.task<Sync>("buildForXcodeDSYM") {
```

Comment:
could you please use the DSL API to find the task instead of hard-coding it's name

Reply:
Not sure I understand. We're not finding an existing task, we're creating a new one.